### PR TITLE
gui-libs/wf-config: fix building with gcc-10

### PR DESCRIPTION
**Pull request by epsilon-0**
due to version bump, also fix its revdep dependency versions

### CODE REVIEW

File: gui-wm/wayfire/wayfire-0.4.0-r2.ebuild [hunk 1]
```diff
@@ -36,7 +36,7 @@ DEPEND="
 	x11-libs/libxkbcommon:=[X]
 	x11-libs/pixman
 	gles? ( media-libs/libglvnd[X] )
-	system-wfconfig? ( ~gui-libs/wf-config-${PV} )
+	system-wfconfig? ( >=gui-libs/wf-config-${PV} )
```
Comment (a17r) [inline gui-wm/wayfire/wayfire-0.4.0-r2.ebuild:39]: maybe comment in git summary why this version is being unrestricted?

Comment (epsilon-0) [reply gui-wm/wayfire/wayfire-0.4.0-r2.ebuild:39]: good point.  
Reason is that only one of the packages was not building with `gcc-10` (to my knowledge). Haven't done extensive testing for the other linked packages, nor has upstream made a release for others.

Relating to edit: upstream hasn't made a clear versioning dependency relations, these links were what I put from testing lots of live and versioned packages myself.

Comment (a17r) [reply gui-wm/wayfire/wayfire-0.4.0-r2.ebuild:39]: I see. If these are usually bumped in sync with each other then I guess that change makes sense.

Comment (epsilon-0) [reply gui-wm/wayfire/wayfire-0.4.0-r2.ebuild:39]: Yep, they have been released together atleast for the past 7-8 months, which seemed like a good enough track record. I might ask upstream to also make releases to the other packages, which will let me keep the ~ (and keep the tree saner)

**Pull request closed without merging**